float64 x
float64 y
float64 theta
string[] bssid
string[] ssid
int16[]  rss